### DISCUSSION TITLE
Button Hover Effect after re-enabling

### DISCUSSION THREAD

**Discussion by Wizard0f0zz** [TomSchimansky/CustomTkinter#2099] [Q&A] [answered]
Hi, 
I am new to python and using customtkinter and I like to report a behavior I noticed during one of my projects.
When a button is re-enabled, it loses the hover effect.
It does not work even when we explicitly enable the hover effect.
In the following example, in button1 the hover effect does not work after it is re-enabled. I suspect this is expected behavior.
However, hover effect on button2 continues to work as its configuration state has not changed. 

<code>
import customtkinter as ctk

app = ctk.CTk()
app.geometry("200x200")
app.title("Button Hover")

def disable_n_enable_btn1():
    button1.configure(state='disabled')
    button1.configure(state='enabled') # without explicitly enabling hover.
    #button1.configure(state='enabled', hover=True) # explicitly enabling hover.

def hello():
    print('hello')

button1 = ctk.CTkButton(master=app, text='Test', command=hello)
button1.grid(row=0, column=0, padx=30, pady=30)
button2 = ctk.CTkButton(master=app, text='Disable', command=disable_n_enable_btn1)
button2.grid(row=1, column=0, padx=30, pady=30)

app.mainloop()
</code>

**Comment by Akascape** [comment, marked as answer]
@Wizard0f0zz Actually "enabled" is not the correct term. It should be `normal`

```python
button1.configure(state='normal')
```

**Reply by Wizard0f0zz** [reply]
Silly me!! Thank you.
